public/app/plugins/panel/table/module.tsx

```diff
@@ -104,6 +104,70 @@ export const plugin = new PanelPlugin<Options, FieldConfig>(TablePanel)
           name: 'Hide in table',
           defaultValue: undefined,
           hideFromDefaults: true,
+        })
+        .addBooleanSwitch({
+          path: 'footer.show',
+          category: [footerCategory],
+          name: 'Show table footer',
+          defaultValue: defaultOptions.footer?.show,
+        })
+        .addCustomEditor({
+          id: 'footer.reducer',
+          category: [footerCategory],
+          path: 'footer.reducer',
+          name: 'Calculation',
+          description: 'Choose a reducer function / calculation',
+          editor: standardEditorsRegistry.get('stats-picker').editor,
+          override: standardEditorsRegistry.get('stats-picker').editor,
+          defaultValue: [ReducerID.sum],
+          process: identityOverrideProcessor,
+          showIf: (cfg: FieldConfig) => cfg.footer?.show,
+          shouldApply: () => true,
+        })
+        .addBooleanSwitch({
+          path: 'footer.countRows',
+          category: [footerCategory],
+          name: 'Count rows',
+          description: 'Display a single count for all data rows',
+          defaultValue: defaultOptions.footer?.countRows,
+          showIf: (cfg) => cfg.footer?.reducer?.length === 1 && cfg.footer?.reducer[0] === ReducerID.count,
+        })
+        .addMultiSelect({
+          path: 'footer.fields',
+          category: [footerCategory],
+          name: 'Fields',
+          description: 'Select the fields that should be calculated',
```

Comment:
this doesnt make sense to have any more, since this option is now at the field level and you will enable it for all or just the ones you're interested in.